runtime/src/test/java/org/corfudb/runtime/view/ClusterHealthTest.java

```diff
@@ -0,0 +1,36 @@
+package org.corfudb.runtime.view;
+
+import org.corfudb.runtime.view.ClusterStatusReport.ClusterStatus;
+import org.corfudb.runtime.view.ManagementView.ClusterHealth;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ClusterHealthTest {
+    private final LayoutUtil layoutUtil = new LayoutUtil();
+
+    /**
+     * Get cluster status from a layout with all responsive nodes. Expected result is STABLE status
+     */
+    @Test
+    public void testClusterHealth() {
+        final String server1 = "server1";
+        final String server2 = "server2";
+        final String server3 = "server3";
+
+        final List<String> servers = Arrays.asList(server1, server2, server3);
+        final Layout layout = layoutUtil.getLayout(servers);
+        layout.setUnresponsiveServers(Arrays.asList(server3));
+
+        ClusterHealth clusterHealth = new ClusterHealth();
+        ClusterStatus status = clusterHealth.getClusterHealth(
+                layout, layout.getAllActiveServers()
+        );
+
+        assertThat(status).isEqualTo(ClusterStatus.STABLE);
```

Comment:
This test is failing in CI. Seems timing related? Can it be made reliable?

Reply:
will do it, thank you

Reply:
Sundar, there couldn't be any timing issues (unit test, no external invocations), my test was failing intentionally, I wanted to add more unit tests (that's why I made it failed) when I created this PR and then just forgot to add more tests.
So, I just added more cases for ClusterHealth, now everything is covered